### DISCUSSION TITLE
Font problem in pdf

### DISCUSSION THREAD

**Discussion by Wrumm** [sumatrapdfreader/sumatrapdf#2470] [Q&A] [answered]
Hi,

I have two PCs that are more or less installed similarly. 
If I open certain PDF files with Sumatra from my bank on PC A, everything looks fine. If I open the same files on PC B with Sumatra, some of the text is unreadable because the letters are replaced by strange symbols.
If I open the files with Acrobat Reader on both PCs they are displayed correctly. 

So there must be something wrong in the configuration of PC B that is only effecting Sumatra to display the files/font correctly. Re-installation of Sumatra didn't help and I do not see at least any obvious difference in the Windows Fonts between the two PCs.

Where do I start to investigate the issue?

**Comment by kjk** [comment]
A properly generated PDF should include the font data for non-standard fonts.

Incorrectly generated PDF refer to non-standard fonts only by name. In that case Sumatra tries it's best to find a matching font in Windows.

That might fail, especially for chinese/japanese fonts.

You can see a list of fonts in a PDF using pre-release SumatraPDF (https://www.sumatrapdfreader.org/prerelease), opening PDF and pressing Ctrl-D and then "Get Fonts Info" button.

That will give you list of fonts.

I need to add a way to view internal Sumatra log as that logs info if the PDF file is not correctly generated.

**Comment by frasosatis** [comment, marked as answer]
I had the same problem on some of my files using SumatraPDF 3.3.3-64 bit.
I downloaded and built the current version on GitHub (3.4) and the problem does not seem to exist anymore - the files are now again shown as expected.

**Reply by kjk** [reply]
Yes, it's possible that some improvements since latest released version 3.3.3 improved font matching.

With bugs it's always recommended to test with pre-release version https://www.sumatrapdfreader.org/prerelease

**Comment by Wrumm** [comment]
Thank you for the advice.
I used today's prerelease version v3.4.14378 as 64-bit and portable version and with that the pdfs are displayed correctly on PC B. 
That is great :-), but I still do not see the reason for the problem before. I do not see any obvious difference in the mentions fonts for the two PCs. On both I have no "Helvetica" but a "Helvetica Fett" (fett = bold in German) and I have "Courier New" with the four subtypes "standard, bold, italic, bold-italic" (already translated) and a single Font "Courier Standard". But all on both PCs the same.

The fonts in the files are:
Courier (Type1)
Courier-Bold (Type1)
Helvetica (Type1)
Helvetica-Bold (Type1)

Not sure if it makes sense to investigate this more, since it is working with the prerelease. Any plans when this will be released?

**Reply by kjk** [reply]
I don't know when the next release will be.

Pre-release is more than good enough for daily use. Unless you get a bug that affects you frequently in which case you can open a bug and hopefully I'll fix it promptly.

**Comment by Wrumm** [comment]
Thanks for the help!
